packages/xrpl/test/client.test.ts

```diff
@@ -0,0 +1,54 @@
+import { assert } from 'chai'
+import { Client } from 'xrpl-local'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('Client', () => {
+  it(
+    'Client - implicit server port',
+    () => {
+      // eslint-disable-next-line no-new -- Need to test constructor
+      new Client('wss://s1.ripple.com')
```

Comment:
An alternate syntax you can use to avoid this eslint error is `_` variables - those indicate that the value is not needed. 
```suggestion
      const _ = new Client('wss://s1.ripple.com')
```